cluster/gce/gci/configure-helper-common.sh

```diff
@@ -3136,9 +3149,7 @@ function start-lb-controller {
     prepare-log-file /var/log/glbc.log
     setup-addon-manifests "addons" "cluster-loadbalancing/glbc"
     setup-addon-manifests "addons" "rbac/cluster-loadbalancing/glbc"
-    if [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
-      create-kubeconfig "l7-lb-controller" ${GCE_GLBC_TOKEN} "${PROXY_RESERVED_IP}" "8888" "http"
-    elif [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == "true" ]]; then
+    if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" == "true" ]]; then
```

Comment:
ENABLE_INSECURE_SCALEOUT_CLUSTER_MODE  here too. l7-lb does not need to talk to RP?

Reply:
logic wise, won't fix. naming wise, same as the other naming changes comments